### PR TITLE
Add Github connection section to project page

### PR DESCRIPTION
**Pull request by ericpgreen2**
Contributes to #2109

### CODE REVIEW

File: web-admin/src/routes/[organization]/[project]/-/deployment/+page.svelte [hunk 1]
```diff
@@ -13,7 +16,7 @@
 <div class="flex flex-col items-center">
   <div class="flex space-x-10 border-b border-gray-200 w-full px-12 py-5">
```
Comment (hamilton) [inline web-admin/src/routes/[organization]/[project]/-/deployment/+page.svelte:17]: Would suggest you use grid here, not flex, and explicitly set these column widths according to the mocks. It'll be much more usable and easier to tweak

File: web-admin/src/components/projects/ProjectGithubConnection.svelte [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+          <a
+            href={$proj.data?.project?.githubUrl}
+            class="flex items-center gap-x-1 text-gray-800"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <Github className="inline-block w-4 h-4" />
+            <span
+              class="font-semibold text-[12px] leading-5"
+              style="font-family: Source Code Variable">{prettyGithubRepo}</span
```
Comment (hamilton) [inline web-admin/src/components/projects/ProjectGithubConnection.svelte:36]: you can use `var(--monospace)` here I believe, in case we change it later

**Review by hamilton — APPROVED** [review]
Overall looks good!

We will definitely need to enforce stronger constraints on column sizes, but this PR can merge first & we can address the remaining layout changes in another PR.

I left a couple of small comments. Feel free to merge after addressing

File: web-admin/src/routes/[organization]/[project]/-/deployment/+page.svelte (hunk 1 above)
Comment (ericpgreen2) [reply web-admin/src/routes/[organization]/[project]/-/deployment/+page.svelte:17]: Good call. I'll change the layout in a different PR, but just added a max-width & truncate here, so there's no excessive overflow.

**Pull request merged**